signer/src/bitcoin/validation.rs

```diff
@@ -223,11 +500,11 @@ impl BitcoinWithdrawalOutputError {
 #[derive(Debug, thiserror::Error, PartialEq, Eq, Copy, Clone)]
 pub enum BitcoinSweepErrorMsg {
     /// The error has something to do with the inputs.
-    #[error("deposit error; {0}")]
-    Deposit(#[from] BitcoinDepositInputError),
+    #[error("deposit error ")]
```

Comment:
nit: extra space

```suggestion
    #[error("deposit error")]
```

Reply:
Done in https://github.com/stacks-network/sbtc/pull/849/commits/70b6f91bf8eb7c016f95a8d9a539157e69c7ab8e.